pkgs/by-name/im/immich-machine-learning/package.nix

```diff
@@ -56,12 +57,14 @@ python.pkgs.buildPythonApplication rec {
     ]
     ++ uvicorn.optional-dependencies.standard;
 
-  nativeCheckInputs = with python.pkgs; [
-    httpx
-    pytest-asyncio
-    pytest-mock
-    pytestCheckHook
-  ];
+  nativeCheckInputs =
+    with python.pkgs;
+    [
+      httpx
+      pytest-asyncio
+      pytest-mock
+    ]
+    ++ lib.optionals (stdenv.buildPlatform.system != "aarch64-linux") [ pytestCheckHook ];
```

Comment:
Could you disable checkPhase instead?
Also, please add a comment why we need to disable tests.

```nix
doCheck = stdenv.buildPlatform.system != "aarch64-linux";
```